src/components/util/sortFn.ts

```diff
@@ -1,19 +1,66 @@
 import { SortDirection } from '../../enums/SortDirection';
 
-export const basicSort = (a: any, b: any, sortDirection: SortDirection) => {
-  if (!a) {
+export type SortCriteria<T> = {
+  key: keyof T | ((item: T) => string);
+  direction: SortDirection;
+}
+
+function getValue<T>(item: T, key: keyof T | ((item: T) => any)): any {
+  return typeof key === 'function' ? key(item) : item[key];
+}
+
+function isNumber(num: any): boolean {
+  return !isNaN(num);
+}
+
+export const multisortFn = <T>(criteria: SortCriteria<T>[]): (a: T, b: T) => number => {
+  return (a: T, b: T): number => {
+    for (const { key, direction } of criteria) {
+      const aValue = getValue(a, key);
+      const bValue = getValue(b, key);
+
+      const result = basicSortFn(aValue, bValue, direction);
+      if (result !== 0) {
+        return result;
+      }
+    }
+    return 0;
+  };
+}
+
+export const basicSortFn = (a: any, b: any, sortDirection: SortDirection): number => {
+  if (a === null) {
```

Comment:
@kuka-radovan please could we treat nulls as 0s so they get sorted too? That seems to me more intuitive when I'm changing the order. 

e.g.

```
export const basicSortFn = (aValue: any, bValue: any, sortDirection: SortDirection): number => {

  const a = aValue == null ? 0 : aValue;
  const b = bValue == null ? 0 : bValue;
```

Reply:
@hjkmarshall Yes, the API returns `null` for some records. Even if there is no value for the corresponding there is null value and we render special symbol, but it caused a problem for numerical column values when sorting. If we want to display missing data e.g. as `N/A` or `-`, I cannot mix numeric and string values when sorting.
So in the datamodel I leave null values and the special symbol is added only during rendering. But I have implemented a logic, if you define empty value as 0 it will behave as situation that BE returns 0. So it will sort as you want. It works with any number.

Reply:
@kuka-radovan I replied to the email to this but can't see the message :S

I'm not talking about what we actually display, just about how we sort, and it looks like you're already sorting based on NULLs right below this. So all I'm saying is that we treat NULLs like 0 so they could get sorted too. Let's discuss on Slack, as I tested this and it worked well.

Reply:
Done